axelrod/moran.py

```diff
@@ -1,21 +1,24 @@
 """Implementation of the Moran process on Graphs."""
 
-from collections import Counter
 import random
+from collections import Counter
+from types import FunctionType
+from typing import List, Optional, Set, Tuple
 
 import matplotlib.pyplot as plt
 import numpy as np
 
-from axelrod import DEFAULT_TURNS, Player, Game
+from axelrod import DEFAULT_TURNS, Game, Player
+
 from .deterministic_cache import DeterministicCache
-from .graph import complete_graph, Graph
+from .graph import Graph, complete_graph
 from .match import Match
 from .random_ import randrange
 
-from typing import List, Tuple, Set, Optional
 
-
-def fitness_proportionate_selection(scores: List) -> int:
+def fitness_proportionate_selection(
+    scores: List, fitness_function: FunctionType = None
+) -> int:
     """Randomly selects an individual proportionally to score.
 
     Parameters
```

Comment:
Need to add `fitness_function` to the docstring (similarly for `MoranProcess.__init__`):

I suggest:

```
fitness_function: A function mapping a score to a float.
```

Reply:
ah missed that. Fixing it